backend/fregepoc/analyzers/tests/typescript/test_typescript_avarage_cyclomatic_complexity.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from fregepoc.analyzers.core import AnalyzerFactory
+from fregepoc.analyzers.tests.typescript.constants import (
+    MOCKED_TYPESCRIPT_FILES,
+)
+from fregepoc.repositories.constants import ProgrammingLanguages
+from fregepoc.repositories.factories import RepositoryFileFactory
+
+
+@pytest.mark.django_db
+class TestTypescriptAnalyzerAvarageCyclomaticComplexity:
+    @pytest.mark.parametrize(
+        ["repo_file_params", "expected_avg_cyclomatic_complexity"],
+        [
+            (
+                {"repo_relative_file_path": "bst.ts"},
+                1.0,
+            ),
+            (
+                {"repo_relative_file_path": "empty.ts"},
+                0.0,
+            ),
+            (
+                {"repo_relative_file_path": "bubble_sort.ts"},
+                2.0,
+            ),
+            (
+                {"repo_relative_file_path": "fast_fibbonaci.ts"},
+                2.5,
+            ),
+        ],
+    )
+    def test_count_loc(
```

Comment:
Is there a reason for not including all those tests in a single 'Test Case' class? I think all those test classes could have been merged into one.

Reply:
They probably could be easily merged, however I am following convention present in every other analyzers tests